dot/state/block.go

```diff
@@ -164,6 +164,7 @@ func headerKey(hash common.Hash) []byte {
 }
 
 // headerHashKey = headerHashPrefix + num (uint64 big endian)
+// TODO: Shouldn't we change this block number to uint32?
```

Comment:
Created https://github.com/ChainSafe/gossamer/issues/2436 for it. It would break database storage, that's why I didn't change it a few weeks ago when changing block numbers to use `uint`

```suggestion
```